src/main/java/com/nvidia/spark/rapids/jni/JSONUtils.java

```diff
@@ -23,6 +23,8 @@ public class JSONUtils {
     NativeDepsLoader.loadNativeDeps();
   }
 
+  public static final int MAX_PATH_DEPTH = 32;
```

Comment:
I did not find any usage for this new variable.

Reply:
It will be used in plugin, for max path instructions fallback.